TLM/TLM/UI/Translation.cs

```diff
@@ -12,77 +17,254 @@ namespace TrafficManager.UI {
     using Util;
 
     public class Translation {
-        public static readonly IDictionary<string, string> LanguageLabels;
-        public static readonly IList<string> AvailableLanguageCodes;
+        /// <summary>
+        /// Defines the order of language selector dropdown.
+        /// </summary>
+        public static readonly IList<string> AvailableLanguageCodes
```

Comment:
Why have separate list of `AvailableLanguageCodes` when we can already get that info from `csv_columns_to_locales_` ?

Reply:
The Dictionary does not preserve order. This preserves order. Unless we find some sorting (like A..Z alphabetical will put German first) that looks nice, and then we can rebuild this list at runtime, because Options window will use this array to populate the language selection dropdown.

Reply:
Replaced with `AvailableLanguageCodes.Values().ToList().Sort()`